### DISCUSSION TITLE
Allow user to configure Peloton auto-resistance

### DISCUSSION THREAD

**Discussion by skyfxl** [cagnulein/qdomyos-zwift#831] [Ideas]
Right now, the auto-resistance feature automatically sets a resistance at the bottom of the range. For example, if the instructor suggests a range of 45-60 resistance, the auto-resistance feature sets a resistance of 44. It would be great if users could use the settings to further configure this to instead be at the middle (52) or top end (60) of the range.

**Comment by cagnulein** [comment]
@skyfxl what if you change the difficulty level in the peloton settings in QZ?

**Comment by skyfxl** [comment]
@cagnulein - got it, thanks!
